src/functions/meta-assistant/slack-user-avatar/handler.ts

```diff
@@ -0,0 +1,50 @@
+import { ValidatedEventAPIGatewayProxyEvent } from "@libs/api-gateway";
+import { middyfy } from "@libs/lambda";
+import SlackUtilities from "src/meta-assistant/slack/slack-utils";
+
+/**
+ * Parameters for lambda
+ */
+export interface SlackUsersAvatarsParameters {
+  email: string,
+}
+
+/**
+ * Response schema for lambda
+ */
+export interface Response {
+  image_original: string,
+	image_24: string,
+	image_32: string,
+}
+
+/**
```

Comment:
I have reviewed the spec and suggested changing this to return  all slack users images in a single response rather than us having to request this for every user. Let me know if you think this idea would work or if there are some problems with it.